README.md

```diff
@@ -41,6 +41,22 @@ Full installation instructions are available in [INSTALL.md](https://github.com/
 
 ## Working with data
 
+### Importing
+
+AcousticBrainz provides data dumps that you can import into your own server.
+Latest database dump is available at http://acousticbrainz.org/download. You
+need to download full database dump from this page and use it during database
+initialization:
+
+    $ python manage.py init_db -a <path_to_the_archive>
+
+or import archive after database is created:
+
+    $ python manage.py import_data -a <path_to_the_archive>
```

Comment:
but importing data with duplicate ids will fail.

I'm tending towards having the import create the database itself. If it already exists, you must use `--force` or `--delete` to indicate that you know it will drop and recreate the db